### DISCUSSION TITLE
How to delete an existing instance?

### DISCUSSION THREAD

**Discussion by carlosblanco** [pockethost/pockethost#115] [Q&A] [answered]
I created an instance but I want to delete it. How can I do it?

**Comment by benallfree** [comment, marked as answer]
Hi @carlosblanco, it is not possible to delete an instance at this time. We have it planned in #35.

In the mean time, can I manually delete it for you?

**Reply by carlosblanco** [reply]
Thanks for your answer. I think I can wait until the feature is ready.
